Basic/ClientDriven/Assets/Scripts/NetworkObjectSpawner.cs

```diff
@@ -9,7 +9,7 @@ namespace Unity.Multiplayer.Samples.ClientDriven
     /// <remarks>
     /// A NetworkManager is expected to be part of the scene that this NetworkObject is a part of.
     /// </remarks>
-    internal class NetworkObjectSpawner : MonoBehaviour
```

Comment:
why are you removing the internal?

Reply:
It's redundant for all class types, and per the standards doc:

> Drop redundant access specifiers (leave off 'internal' at file scope)

Reply:
interesting good to know! Where can I find the standards doc?

Reply:
Sent it offline!